eliot/_action.py

```diff
@@ -447,25 +490,25 @@ def context(self):
 
         The action does NOT finish when the context is exited.
         """
-        _context.push(self)
+        _context_owner.context.push(self)
         try:
             yield self
         finally:
-            _context.pop()
+            _context_owner.context.pop()
 
     # Python context manager implementation:
     def __enter__(self):
         """
         Push this action onto the execution context.
         """
-        _context.push(self)
+        _context_owner.context.push(self)
         return self
 
     def __exit__(self, type, exception, traceback):
         """
         Pop this action off the execution context, log finish message.
         """
-        _context.pop()
+        _context_owner.context.pop()
```

Comment:
This set of changes, from `_context` to `_context_owner.context` makes sense.  The current execution context object state has moved from the module global to the `_ECOwner` instance attribute.  Great.